tests/integration/builders/builders_test.go

```
@@ -0,0 +1,258 @@
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
+			g := NewWithT(t)
+
+			tmpFile, err := os.CreateTemp(t.TempDir(), "Dockerfile-*")
+			g.Expect(err).NotTo(HaveOccurred())
+
+			_, err = tmpFile.WriteString(tc.dockerfileContent)
+			g.Expect(err).NotTo(HaveOccurred())
+			tmpFile.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `tmpFile.Close` is not checked

(errcheck)